src/Components/Blazor/Build/test/BuildIntegrationTests/BuildIntegrationTest.cs

```diff
@@ -26,6 +26,9 @@ public async Task Build_WithDefaultSettings_Works()
             Assert.FileExists(result, buildOutputDirectory, "dist", "_framework", "wasm", "dotnet.js");
             Assert.FileExists(result, buildOutputDirectory, "dist", "_framework", "_bin", "standalone.dll");
             Assert.FileExists(result, buildOutputDirectory, "dist", "_framework", "_bin", "Microsoft.Extensions.Logging.Abstractions.dll"); // Verify dependencies are part of the output.
+
+            var staticWebAssets = Assert.FileExists(result, buildOutputDirectory, "standalone.StaticWebAssets.xml");
+            Assert.FileContains(result, staticWebAssets, @"netstandard2.1\dist\".Replace('\\', Path.DirectorySeparatorChar));
```

Comment:
Nit: consider using Path.Combine instead of `.Replace('\\', Path.DirectorySeparatorChar)`

Reply:
Also below